xml/System.Windows.Forms/HighDpiMode.xml

```diff
@@ -74,7 +74,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The window checks for DPI when it's created and adjusts scale factor when the DPI changes.</summary>
```

Comment:
Can we word it to get following  captured?

Changing DPI while application is running does not impact scaling. DPI may be changed when application is moved from one monitor to the other ( if there  are more than one monitor attached and their DPI settings were different).  in this case application rescale according to new DPI of the monitor.

Reply:
I would remove this sentence:
>Changing DPI while application is running does not impact scaling

it's possible to change DPI scaling on a single screen and applications with PerMonitor (v1) will get the WM_DPICHANGED message....

Reply:
@dreddy-work your suggestion would look better in the Remarks section. Here is the suggestion (with some rewording) and excluding the sentence mentioned by @Tanya-Solyanik . Can you please let me know if you agree?

@mairaw feel free to edit my suggestion if necessary:

```suggestion
        <summary>The window checks for DPI when it's created and adjusts scale factor when the DPI changes.</summary>
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

If there is more than one monitor attached, and their DPI settings are different, the DPI may be changed when the application is moved from one monitor to the other. In this case, the application is rescaled according to the new monitor's DPI settings.

          ]]></format>
        </remarks>
```

Reply:
@rpetrusha you can also edit my suggestion if necessary.

Reply:
I would add that `Alternatively DPI of a window can be changed when OS scaling settings are changed for the monitor the  window is on.`

Reply:
I would say 'window' not 'application'. DPIMode applies to application or process, but DPI value applies to a specific window.
`the DPI may be changed when the __window__ is moved from one monitor `


If there is more than one monitor attached, and their DPI settings are different, the DPI may be changed when the application is moved from one monitor to the other. In this case, the application is rescaled according to the new monitor's DPI settings.
 





If there is more than one monitor attached, and their DPI settings are different, the DPI may be changed when the application is moved from one monitor to the other. In this case, the application is rescaled according to the new monitor's DPI settings.
 

Reply:
Thanks @Tanya-Solyanik , I added your extra sentence, and modified application->window. Can you take a look to the suggestion, and let us know if you agree?

Reply:
@mairaw if the suggestion looks good, please feel free to commit it.